### ISSUE TITLE
[Roadmap] Define Flow, Instructions & Tools to present E2E demos

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/pocket#350]
## Objective

Enable anyone (technical or not) to execute each iteration's E2E demos.

## Origin Document

Not blocked by, but heavily related to #349.

## Goals

- Provide instructions on how to perform E2E demos

## Deliverable

- [ ] Document the steps (w/ copy-pastable commands) on how to execute the demos
- [ ] If needed, very small tools, makefile commands or scripts to ease testing

## Non-goals / Non-deliverables

- Making any modification to the pocket core business logic

## General issue deliverables

- [ ] Update the appropriate CHANGELOG
- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references

## Testing Methodology

- Testing will be manual with "human checks" at different points
- **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)

---

**Creators**: @Olshansk @jessicadaugherty

**Comment by Olshansk** [comment]
First approach attempt at something for iteration 3 can be found here:
* Branch: https://github.com/pokt-network/pocket/tree/iteration3_demo
* Demo: https://drive.google.com/file/d/1IOrzq-XJP04BJjyqPPpPu873aSfwrnur/view?usp=share_link
* Instructions:https://github.com/pokt-network/pocket/blob/iteration3_demo/iter_3_demo_instructions.md

**Comment by Olshansk** [comment]
We will use this document as a baseline for future similar flows: https://github.com/pokt-network/pocket/blob/main/docs/demos/iteration_3_end_to_end_tx.md

Closing this for now as it is otherwise "endless"

**Comment by jessicadaugherty** [comment]
Thank you @Olshansk!
